rust/protocol/src/identity_key.rs

```diff
@@ -1,8 +1,14 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2020-2022 Signal Messenger, LLC.
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
+//! Wrappers over cryptographic primitives from [`crate::curve`] to represent [identity].
+//!
+//! [identity]: https://en.wikipedia.org/wiki/Online_identity
```

Comment:
This is real abstract, heh. Maybe we can get away with just saying "represent a user"?

Reply:
Done!

Reply:
I had been heavily editing the wikipedia page on "privacy" around this time last year and got some wires crossed, I think.